tests/bud.bats

```diff
@@ -2099,6 +2099,27 @@ _EOF
   done
 }
 
+@test "bud and test --unsetlabel" {
```

Comment:
_prefetch alpine here.

Reply:
the base is actually `fedora:minimal` since it contains lot of pre-loaded labels for this test, shouldn't this be `_prefetch fedora:minimal`